### PR TITLE
Development

### PR DESCRIPTION
**Pull request by redfrexx**
* Added dependecies for development to readme.
* Fixed creation of request url 
* Removed error code from log file name

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,13 +9,34 @@ The ohsome API provides various endpoints for [data aggregation](https://api.ohs
...
 
-If you want to run the Juypter Notebook [Tutorial](https://github.com/GIScience/ohsome-py/blob/master/notebooks/Tutorial.ipynb) you also need to install `jupyter` and `matplotlib`:
+### Using Anaconda
+
+*ohsome-py* is not available through Anaconda yet. So if you are using Anaconda, create a new anaconda environment and install the required dependencies before installing *ohsome-py* using pip.
+
+```
+$ conda create -n ohsome python=3.8 geopandas requests
+$ conda activate ohsome
+$ pip install ohsome --no-deps
```
Comment (SlowMo24) [inline README.md:34]: should we add a note here? I mean anaconda and pip do not work nicely together so this is non of our business... How about we just link to the offical documentation or say `be aware that pip my seriously damage your conda environment` https://www.anaconda.com/blog/using-pip-in-a-conda-environment

File: README.md [hunk 2]
```diff
@@ -9,13 +9,34 @@ The ohsome API provides various endpoints for [data aggregation](https://api.ohs
 
 ## Installation
 
-*ohsome-py* requires Python >= 3.6. The easiest way to install *ohsome-py* is using pip:
+*ohsome-py* requires
+
+* Python >= 3.6
+* geopandas >= 0.9.0
```
Comment (SlowMo24) [inline README.md:15]: wouldn't goepandas 0.8.0 be sufficient? thats what conda currently has (I#m not advocating for conda, all of these env-managers are cumbersome, just trying to make it the most simple for all users)

File: README.md [hunk 3]
```diff
@@ -126,7 +147,17 @@ time = pandas.date_range("2018-01-01", periods=3, freq="M")
 
 If you want to contribute to this project, please fork the repository or create a new branch containing your changes.
 
-**Install the pre-commit hooks** in our local git repo before commiting to ensure homogenous code style.
+**Install dependencies for development**
```
Comment (SlowMo24) [inline README.md:150]: sorry, again conda: do you want to mark which ones are not available there?

File: ohsome/helper.py [hunk 4]
```diff
@@ -175,19 +177,50 @@ def find_groupby_names(url):
...
+
+    m = re.search('"path" : "(.*)"', responsetext)
+    if m:
+        message += "; " + m.group(0)
+
+    m = re.search('"requestUrl" : "(.*)"', responsetext)
+    if m:
+        message += "; " + m.group(0)
+
+    m = re.search(r'"status" : (\d+|".*")', responsetext)
```
Comment (SlowMo24) [inline ohsome/helper.py:212]: such regex, much wow :-)

**Review by SlowMo24 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (redfrexx) [reply README.md:15]: Yes, I thought the same, but for some reason some test fail with geopandas 0.8.0. So I'd leave it like that and test it again when I have time.

Comment (redfrexx) [reply README.md:15]: Yes, I thought the same, but for some reason some test fail with geopandas 0.8.0. So I'd leave it like that and test it again when I have time.

File: README.md (hunk 3 above)
Comment (redfrexx) [reply README.md:150]: yes, good idea. Will add it to the readme.

File: README.md (hunk 1 above)
Comment (redfrexx) [reply README.md:34]: yes, I note would be good. I'll add it.

File: ohsome/helper.py (hunk 4 above)
Comment (redfrexx) [reply ohsome/helper.py:212]: ^^

**Pull request merged**